### PR TITLE
feat(shared): split downloadButton out of the Attachment

### PR DESCRIPTION
**Pull request by ragi96**
Splitting the downloadButton out of the attachments components so the attachments component has a single purpose.

There will be a follow-up PR with adjusting the Props of the attachment

### CODE REVIEW

File: packages/shared/src/components/DownloadButton/DownloadButton.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    return (
+        <>
+            <button
+                data-test-id="download-button"
+                onClick={onDownload}
+                className="tw-rounded-full tw-mb-auto tw-bg-box-neutral-strong-inverse hover:tw-bg-box-neutral-strong-inverse-hover active:tw-bg-box-neutral-strong-inverse-pressed  tw-text-box-neutral-strong tw-outline tw-outline-1 tw-outline-offset-[1px] tw-p-1.5 tw-outline-line"
+            >
+                <IconArrowCircleDown16 />
+            </button>
+        </>
```
Comment (celine-s) [inline packages/shared/src/components/DownloadButton/DownloadButton.tsx:17]:
```suggestion
          <button
              data-test-id="download-button"
              onClick={onDownload}
              className="tw-rounded-full tw-mb-auto tw-bg-box-neutral-strong-inverse hover:tw-bg-box-neutral-strong-inverse-hover active:tw-bg-box-neutral-strong-inverse-pressed  tw-text-box-neutral-strong tw-outline tw-outline-1 tw-outline-offset-[1px] tw-p-1.5 tw-outline-line"
          >
              <IconArrowCircleDown16 />
          </button>
```

**Review by celine-s — COMMENTED** [review]
Nice! love it! just one small comment. and I would rename the PR to `feat(shared): split downloadButton out of the Attachment`

File: packages/shared/src/components/DownloadButton/DownloadButton.tsx (hunk 1 above)
Comment (ragi96) [reply packages/shared/src/components/DownloadButton/DownloadButton.tsx]: removed the fragment around, thanks

**Review by celine-s — APPROVED** [review]
🚀

**Review by fulopdaniel — APPROVED** [review]

**Pull request merged**